### ISSUE TITLE
missing <media:content> tag

### ISSUE THREAD

**Issue by bubudrc** [brentsimmons/RSParser#70]
parsing the feed: https://www.lanacion.com.ar/arcio/rss/ I'm found that the tag **<media:content>** is not taking account. Are there any way to inspect that property after to do `FeedParser.parse(...)` on the **items** value?

Thanks

**Comment by brentsimmons** [comment]
Good point. We don’t deal with enclosures much, since this is tuned to the needs of NetNewsWire. We probably should handle enclosures, though.

**Comment by bubudrc** [comment]
ok. I will try to create a PR to improve this behavior :)

**Comment by bubudrc** [comment]
open a PR: https://github.com/Ranchero-Software/RSParser/pull/71 for this issue
